plenty_api/api.py

```diff
@@ -1235,6 +1237,31 @@ def plenty_api_create_booking(self, order_id: int,
                                              data=data)
         return response
 
+
+    def plenty_api_create_property_selection_name(self, json: dict) -> dict:
+        """
+        Create a language for a selection property.
+
+        Parameter:
+            json            [dict]  -   single Json object describing the
+                                        creation
+
+        Return:
+                            [dict]  -   Response object if the request should
+                                        fail, the entry contains the error
```

Comment:
This sounds like you return the response object only if the request fails.
How about:
```
Return the response object, in case the request fails return an error message
```